superset-frontend/spec/javascripts/views/CRUD/data/database/DatabaseList_spec.jsx

```diff
@@ -178,3 +182,44 @@ describe('DatabaseList', () => {
     );
   });
 });
+
+describe('RTL', () => {
+  async function renderAndWait() {
+    const mounted = act(async () => {
+      render(
+        <QueryParamProvider>
```

Comment:
When using render from spec/helpers/testing-library the theme provider is automatically included and you can also request the redux provider, the drag and drop provider, and also initialize the redux state if you want.

I created a PR https://github.com/apache/superset/pull/14195 so you can also activate the `QueryParamProvider`. After the merge you can do:
`render(<DatabaseList user={mockUser} />, { useRedux: true, useQueryParams: true });`

😉

Reply:
Awesome, this is going to be much cleaner! Thank you!